tc_aws/aws/session.py

```diff
@@ -1,11 +1,14 @@
 import botocore.session
+from botocore.utils import fix_s3_host
 
+__all__ = ['get_session']
 
-session = None
+sessions = [None, None]
 
-
-def get_session():
-    global session
+def get_session(endpoint=None):
+    session = sessions[bool(endpoint)]
```

Comment:
Hmm, you're right. Its only possible to use standard S3, or a single custom endpoint, not both at the same time.

I'll make the change on Monday.